semana19/cookenu-projeto/src/services/Authenticator.ts

```diff
@@ -0,0 +1,28 @@
+import { sign, verify}  from "jsonwebtoken";
+import { AuthenticationData } from "../types";
+import { config } from "dotenv"
+
+config();
+//gerar o token
+export class Authenticator { 
+   generateToken = (info:AuthenticationData): string => sign(
+            {info},
+            process.env.secret!,
+            {expiresIn: "7d"}
+           
```

Comment:
dica: use o .env EXPIRES_IN = 7d. Assim quem for testar a sua aplicação encontra todas as variáveis de ambientes necessárias para controlar a aplicação.